client/src/components/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {shallow, mount} from 'enzyme';
+import App from './App.jsx';
+
+
+it('App renders without crashing', () => {
+ shallow(<App />);
+});
```

Comment:
Glad to see it works. I am thinking that it could be better to have more tests for more edge test cases.

Reply:
Other than that, I think this is good to go merge